parse.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+source open_parser_base.sh
+
+# Check if number of arguments is not equal to 3
```

Comment:
Update comment to `less than 2`

Reply:
I'll just remove the comment to make it consistent w/ `extract_parse.sh`